docs/human-interface-guidelines.md

```diff
@@ -641,11 +641,11 @@ See also: _[3 Ways to Make Checkboxes, Radio Buttons Easier to Click](http://uxm
 
 Notebooks are a type of widget that allow showing one of multiple pages in an app, also colloquially referred to as "tab bars."
 
-### Static Notebook {#static-notebook}
+### Stack Switcher (former Static Notebook) {#static-notebook}
```

Comment:
This should go from `{#static-notebook}` to `{#stack-switcher}`, and check for other links to that.